pkgs/by-name/ya/yara-x/package.nix

```
@@ -0,0 +1,44 @@
+{ lib
+, fetchFromGitHub
+, rustPlatform
+, cmake
+, installShellFiles
+, testers
+, yara-x
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "yara-x";
+  version = "0.3.0";
+
+  src = fetchFromGitHub {
+    owner = "VirusTotal";
+    repo = "yara-x";
+    rev = "v${version}";
+    hash = "sha256-AFRKsBNDgjIsLnr61ME4WvDj+DsvIhFUnDmOGn3So8o=";
+  };
+
+  cargoHash = "sha256-cZ/bWaTNnX9+o8D5lMu72snc4CLpjqcwjintbw59OXA=";
+
+  nativeBuildInputs = [ cmake installShellFiles ];
+
+  postInstall = ''
+    installShellCompletion --cmd yr \
+      --bash <($out/bin/yr completion bash) \
+      --fish <($out/bin/yr completion fish) \
+      --zsh <($out/bin/yr completion zsh)
+  '';
+
+  passthru.tests.version = testers.testVersion {
+    package = yara-x;
+  };
+
+  meta = {
+    description = "The pattern matching swiss knife for malware researchers, and everyone else. A rewrite of YARA in Rust.";
```

```suggestion
    description = "Tool to do pattern matching for malware research";
```

see https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#meta-attributes about the period.